src/seedu/addressbook/AddressBook.java

```diff
@@ -308,11 +308,20 @@ private static void setupDefaultFileForStorage() {
      * Returns true if the given file is acceptable.
      * The file path is acceptable if it ends in '.txt'
      * TODO: Implement a more rigorous validity checking.
+     * Note: Access modifier was changed from private to 
+     * protected so as to test the method.
```

Comment:
Comments should describe the current state rather than what happened in the past. So it should explain why the method is protected instead of private. The fact that it was changed from private to protected is not relevant.